database/sqlite/sqlite_aclk_alert.c

```diff
@@ -433,8 +430,8 @@ void sql_queue_existing_alerts_to_aclk(RRDHOST *host)
 
     buffer_free(sql);
 
-    if (host->dbsync_worker) {
-        struct aclk_database_worker_config *wc  = (struct aclk_database_worker_config *)host->dbsync_worker;
+    struct aclk_database_worker_config *wc  = (struct aclk_database_worker_config *)host->dbsync_worker;
+    if (wc) {
         __sync_synchronize();
```

Comment:
Not needed before and after this single assignment

Reply:
removed, ok.